brut.apktool/apktool-lib/src/main/java/brut/androlib/res/AndrolibResources.java

```diff
@@ -426,10 +449,9 @@ private void aapt2Package(File apkFile, File manifest, File resDir, File rawDir,
         }
 
         if (apkOptions.doNotCompress != null) {
-            for (String file : apkOptions.doNotCompress) {
-                cmd.add("-0");
-                cmd.add(file);
-            }
+            String extensionsFilePath = createDoNotCompressExtensionsFile(apkOptions).getAbsolutePath();
```

Comment:
See here how we remove the old method. Should we instead do the following?

If customAppt is being used --> old method
If nativeAppt is being used --> new method

So people loading their own aapt binaries get the old method, which works pretty well unless on Windows and I'm pretty sure anyone using a custom aapt binary is probably scripting it on unix.

Reply:
hmm ignore this, I have a tiny refactor to do. I'll take it from here.